.github/workflows/master_only.yml

```diff
@@ -120,6 +110,11 @@ jobs:
         run: pip install pip-tools
       - name: Install dependencies
         run: make install-python-ci-dependencies
+      - name: Start Redis
+        uses: supercharge/redis-github-action@1.4.0
+        with:
+          redis-version: ${{ matrix.redis-version }}
+          redis-port: 12345
```

Comment:
Are the tests configured to use this port for redis?

Reply:
Yup, redis in master is using the ports but we didn't update mater_only.yml so we need to do that for integration tests in master to work.